kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -452,15 +456,15 @@ static Condition resolvedRefsUnknown(Clock clock,
         return newUnknownCondition(clock, observedGenerationSource, Condition.Type.ResolvedRefs, e);
     }
 
-    static String slug(String singular, String group, String name) {
-        return singular + "." + group + "/" + name;
+    public static @NonNull String namespacedSlug(LocalRef<?> ref, HasMetadata resource) {
```

Comment:
I suppose we should call this one `slug`, because namespaces are standard, and the private one could be `localSlug`.